index.html

```diff
@@ -220,6 +219,95 @@ <h3>Skills</h3>
 			</div>
 		</section>
 
-		<script src="main.js"></script>
-	</body>
-</html>
+        <section id="contact">
+			<div class="pri-text">
+				<p class="contact-para">
+					I'm always interested in hearing about new projects, so if you'd like
+					to chat please get in touch.
+				</p>
+			</div>
+			<form action="https://formspree.io/f/xpznwejk" method="post" class="form">
+				<fieldset>
+					<label class="firstN"
+						><input
+							type="text"
+							name="firstname"
+							id="firstname"
+							maxlength="30"
+							placeholder="First name"
+							required
+						/>
+					</label>
+					<label class="lastN">
+						<input
+							type="text"
+							name="lastname"
+							id="lastname"
+							maxlength="30"
+							placeholder="Last Name"
+							required
+						/>
+					</label>
+					<label class="fullN"
+						><input
+							type="text"
+							name="fullname"
+							id="fullname"
+							maxlength="30"
+							placeholder="Full name"
+							required
+						/>
+					</label>
+
+					<label class="email"
+						><input
+							type="email"
+							name="email"
+							id="email"
+							placeholder="Email address"
+							required
+						/>
+					</label>
+					<label class="msg">
+						<textarea
+							name="msg"
+							id="msg"
+							cols="30"
+							rows="10"
+							maxlength="500"
+							minlength="30"
+							placeholder="Enter text here"
+							required
+						></textarea>
+					</label>
+					<button class="btn" type="submit">Get in Touch</button>
+				</fieldset>
+			</form>
+		</section>
+        <footer id="copyright">
+			<div class="copyright">
+				<ul class="socials">
+					<li>
+						<a href=""><i class="fa-brands fa-github"></i></a>
+					</li>
+					<li>
+						<a href=""><i class="fa-brands fa-linkedin-in"></i></a>
+					</li>
+					<li>
+						<a href=""><img src="../assets/Vector.png" alt="vector" /></a>
+					</li>
+					<li>
+						<a href=""><i class="fa-brands fa-twitter"></i></a>
+					</li>
+					<li>
+						<a href=""><i class="fa-solid fa-m"></i></a>
+					</li>
+				</ul>
+				<p class="copyright-txt">myemailhere@email.com</p>
+			</div>
+		</footer>
+        
+
+    <script src="main.js"></script>
```

Comment:
- [ ] Excellent job adding the additional feature. 👍👍 Very well done. However, as mentioned in the requirement, you don't need to implement any functionality that requires JavaScript. I appreciate your effort 🥳. Following is a quote for related requirement

  > You don't need to implement any functionality that requires JavaScript.

**_Please keep this in mind and note that this part of your project is not reviewed because you have submitted your project for  **_Portfolio: desktop version: day 2_** Please keep this in mind when you will be asked to implement JavaScript for mobile menu_**

**_Remember to provide a link to the additional code in the Pull Request created for the next project. For example: you can create a link(s) to specific commit(s) and include that link in the description of Pull Request for your next project_**